src/main/java/org/sonarlint/intellij/actions/SonarLintToolWindow.java

```diff
@@ -377,6 +377,34 @@ public boolean doesSecurityHotspotExist(String securityHotspotKey) {
     return false;
   }
 
+  public void trySelectIssueForCodeFix(String findingKey) {
+    var toolWindow = getToolWindow();
+    if (toolWindow != null) {
+      var contentManager = toolWindow.getContentManager();
+      var content = contentManager.findContent(SonarLintToolWindowFactory.CURRENT_FILE_TAB_TITLE);
+      if (content != null) {
+        var currentFilePanel = (CurrentFilePanel) content.getComponent();
+        var issue = currentFilePanel.doesIssueExistFiltered(findingKey);
```

Comment:
Given its name, I was expecting this `doesIssueExistFiltered` method to return a boolean. Maybe just getIssueFiltered?